README.md

```diff
@@ -7,7 +7,7 @@
 
 # github-actions-runner-operator
 
-K8S operator for scheduling github actions runner pods.
+K8s operator for scheduling github actions runner pods.
```

Comment:
```suggestion
k8s operator for scheduling github actions runner pods.
```

Reply:
"Kubernetes (K8s) is an open-source system for automating deployment, scaling, and management of containerized applications." 
https://kubernetes.io/

The first letter gets capitalized when it's at the beginning of a sentence.